.github/scripts/trymerge.py

```diff
@@ -2340,9 +2358,10 @@ def handle_exception(e: Exception, title: str = "Merge failed") -> None:
         else:
             print("Missing comment ID or PR number, couldn't upload to Rockset")
     finally:
-        gh_remove_label(
-            org, project, args.pr_num, MERGE_IN_PROGRESS_LABEL, args.dry_run
-        )
+        if not args.check_mergeability:
```

Comment:
From what I see, it would always reach this part with `not args.check_mergeability`, so the if statement is kind of redundant?

Reply:
Good catch. For some reason I thought the `try` block starts earlier.